### PR TITLE
Renames Butt to Ass

### PR DESCRIPTION
**Pull request by PopNotes**
:cl: PopNotes
spellcheck: Butt = ass.
/:cl:

### CODE REVIEW

**Comment by Blute1** [comment]
name your PR correctly

**Comment by PopNotes** [comment]
#4402
#4334

**Comment by GuyonBroadway** [comment]
Buttbot is a time honoured tradition.

**Comment by GuyonBroadway** [comment]
If I were Anal about it I would ctrl+f and change every ass to arse.

**Comment by banthebantz** [comment]
no -1

File: hippiestation/code/modules/mob/living/simple_animal/bot/buttbot.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /mob/living/simple_animal/bot/buttbot
-	name = "buttbot"
-	desc = "It's a robotic butt. Are you dense or something??"
+	name = "assbot"
```
Comment (ghost) [inline hippiestation/code/modules/mob/living/simple_animal/bot/buttbot.dm:2]: No.

**Comment by dreamcrab** [comment]
ass update =6

**Comment by Pyko1** [comment]
someone might confuse a butt in to a domesticated donkey with this PR merged!

**Comment by McBawbaggings** [comment]
no
butt is funnier than ass

**Comment by Pyko1** [comment]
I mean this PR is perverted and vile!
Someone farting and blowing his butt off is fine.

But we don't want people farting and blowing off a domesticated donkey...

**Comment by JohnGinnane** [comment]
Butt is more childish and fun

**Comment by tortellinitony** [comment]
NO! FRICK OFF

**Comment by Dudeinyourbackyard** [comment]
butt is best
-1

**Comment by Chnkr** [comment]
unlikely to get merged, -2 head votes, closing pr

**Pull request closed without merging**